DrawerKit/DrawerKit/Internal API/PresentationController.swift

```diff
@@ -155,4 +155,15 @@ extension PresentationController {
     }
 }
 
-extension PresentationController: DrawerPresentationControlling {}
+extension PresentationController: DrawerPresentationControlling {
+    public func setDrawerState(_ state: DrawerState, animated: Bool, completion: (() -> Void)?) {
+        switch state {
+        case .transitioning:
+            fatalError("`transitioning` state is not allowed")
+        case .dismissed:
+            presentedViewController.dismiss(animated: animated, completion: completion)
+        default:
+            animateTransition(to: state, animateAlongside: nil, completion: completion)
```

Comment:
I don't think passing `nil` to `animateAlongside` is the right thing to do because the presenting and presented VCs may have animations that they want to run alongside the drawer animations and they need to be "in sync" with the drawer. If the user performs a drawer drag and those VCs do their animations alongside, and then the client code programmatically animates the drawer to another state, using `nil` for `animateAlongside`, the next time the user drags the drawer, the VC animations will be totally out of sync with the state of the drawer.